pyquil/gates.py

```diff
@@ -268,6 +268,23 @@ def RZ(angle: ParameterDesignator, qubit: QubitDesignator) -> Gate:
     return Gate(name="RZ", params=[angle], qubits=[unpack_qubit(qubit)])
 
 
+def U(theta: ParameterDesignator, phi: ParameterDesignator, lam: ParameterDesignator, qubit: QubitDesignator) -> Gate:
+    """Produces the RZ gate::
```

Comment:
> Produces the RZ gate

Is that the correct description? Doesn't it also produce RX and RY?

Reply:
Yuup, thanks.